### ISSUE TITLE
(MyXQL.Error) (1835) (ER_MALFORMED_PACKET) Malformed communication packet with Mariadb 10.3.26

### ISSUE THREAD

**Issue by Efesto** [elixir-ecto/myxql#127]
by pulling a fresh mariadb image
```
docker-compose -f docker-compose.yml up -d mariadb-10.3
Pulling mariadb-10.3 (mariadb:10.3)...
10.3: Pulling from library/mariadb
6a5697faee43: Already exists
ba13d3bc422b: Already exists
a254829d9e55: Already exists
2ee2cadd29fc: Already exists
6915a184049d: Already exists
5ca6ffdb5f56: Already exists
1537f7bbef8b: Already exists
5790e54322d1: Already exists
ea98cb829471: Already exists
d91f40693863: Already exists
13f0c0d55992: Already exists
6bfbacb89af2: Already exists
c1aebda0a191: Already exists
Digest: sha256:5851f6fc268c9c9188a1350cf244aca5938607a8e29738054daa53e2deab7d4e
Status: Downloaded newer image for mariadb:10.3
Starting myxql_mariadb-10.3_1 ... done
```

and running `scripts/test-versions.sh mariadb-10.3` I get numerous red tests with `(MyXQL.Error) (1835) (ER_MALFORMED_PACKET) Malformed communication packet`

This problem seems to be related to https://jira.mariadb.org/browse/MDEV-23989 and looks the same cause behind https://jira.mariadb.org/browse/MDEV-24134 and https://jira.mariadb.org/browse/MDEV-24121

Everything works fine with 10.3.25
I didn't test with other minor versions

**Comment by wojtekmach** [comment]
Thank you for the great report & follow-up links, I was able to reproduce this locally and will work on a fix.

**Comment by wojtekmach** [comment]
The fix is out in v0.4.4, thanks again!

**Comment by Efesto** [comment]
@wojtekmach that was damn fast! Thank you!
